middlewares/osio/auth.go

```diff
@@ -20,10 +20,12 @@ const (
 type RequestType string
 
 const (
-	api     RequestType = "api"
-	metrics RequestType = "metrics"
-	console RequestType = "console"
-	logs    RequestType = "logs"
+	api      RequestType = "api"
+	oapi     RequestType = "oapi"
+	metrics  RequestType = "metrics"
+	console  RequestType = "console"
+	logs     RequestType = "logs"
+	undefine RequestType = ""
 )
```

Comment:
`oapi` type added for '/oapi' paths.  now, rest all path will be referred as `undefine` type.